src/library.js

```diff
@@ -3226,12 +3226,14 @@ LibraryManager.library = {
 #endif
   },
 
+#if !WASM_BACKEND
```

Comment:
Looks like sched_yield.c is always included so why not just remove this completely?

Reply:
I saw a test failure with fastcomp without that. I'm not sure why fastcomp might use it - I don't see references from JS that could cause that. So my guess is something with how fastcomp links, there might actually be a bug there. But I didn't investigate as it's deprecated.